### DISCUSSION TITLE
Co-working with `indant-blankline.nvim`

### DISCUSSION THREAD

**Discussion by nyngwang** [shellRaining/hlchunk.nvim#22] [Show and tell]
By setting `horizontal_line`, `vertical_line`, `left_bottom`, and `right_arrow` all to `''`, we can keep the indent line of the current chunk created by `indant-blankline.nvim` while still having the benefit of `hlchunk.nvim`: easily find the "header" of the current chunk. Anyway, just want to share this. (Sincerely, I was thinking about removing the former one due to the complexity of my config.)

https://user-images.githubusercontent.com/24765272/236288271-98fe85d4-816b-4afe-a0c5-3b02a173a3fd.mov

<details>
<summary>Click to expand the config</summary>

```lua
use {
  'shellRaining/hlchunk.nvim',
  config = function ()
    require('hlchunk').setup {
      chunk = {
        enable = true,
        use_treesitter = true,
        chars = {
          horizontal_line = "",
          left_top = "󱩡", -- or change it to any icon you like.
          vertical_line = "",
          left_bottom = "",
          right_arrow = "",
        },
        style = "#ffc2e2",
      },
      line_num = {
        enable = true,
        use_treesitter = true,
        style = "#adb5bd",
      },
      blank = { enable = false },
      indent = { enable = false },
    }
    vim.api.nvim_create_autocmd({ 'ColorScheme' }, {
      -- group = curfile_augroup,
      pattern = '*',
      callback = function ()
        pcall(vim.cmd, 'EnableHLChunk')
        pcall(vim.cmd, 'EnableHLLineNum')
      end
    })
  end
}
```

</details>

**Comment by shellRaining** [comment]
Great idea, I will implement it in future versions.

**Reply by nyngwang** [reply]
While it might be too late: the symbol in the provided config is what you were wondering [there](https://github.com/shellRaining/hlchunk.nvim/issues/23#issuecomment-1537135409).

Regarding the implementation, it's currently working well after #18 was resolved. Since only the `top-left` option is needed in my specific case here, I would expect that the implementation could be simplified a lot, and maybe we can get some performance boost.

Sincerely thanks for your help!
